test/functional/readconcern_tests.js

```diff
@@ -56,6 +56,52 @@ describe('ReadConcern', function() {
     }
   });
 
+  it('Should set local readConcern on db level when using createCollection method', {
+    metadata: { requires: { topology: 'replicaset', mongodb: '>= 3.2' } },
+
+    test: function(done) {
+      var listener = require('../..').instrument(function(err) {
```

Comment:
I know you got this from the test above, but let's take this opportunity to modernize this test by changing all `var`s to `let`s or `const`s, by using arrow functions instead of `function(err)`s, and by using `expect` instead of `test.equal`.

Reply:
Let's also replace the `listener` API with the modern APM API:

1. remove the `var listener = require('../..').instrument(...)`
2. Pass `monitorCommands: true` to the client on creation:
    ```js
    const client = configuration.newClient({ w: 1 }, { poolSize: 1, readConcern: { level: 'local' }, monitorCommands: true });
    ```
3. instead of `listener.on('started' ...);`, use `client.on('commandStarted', ...);`
    ```js
     client.on('commandStarted' , () => {
      if (event.commandName === 'find') {
        started.push(event);
      }
    });
    ```